website/docs/developer/research/research-graphql.md

```diff
@@ -211,12 +217,55 @@ Hasura cloud service is not approved by NGAP so it cannot be used for now. Howev
 - Some examples of the schema and resolvers for this server can be found [here](https://www.apollographql.com/docs/apollo-server/getting-started/).
 - Once created, GraphQL queries can be executed from Apollo Sandbox console.
 
+##### Practical Evaluation
+- Documentation is severely limited, with the section for connecting to databases [requesting additional coders to complete missing functionality](https://www.apollographql.com/docs/tutorial/data-source/).
+- Some tutorials exist, but they are [generally a few years old and out-of-date](https://www.robinwieruch.de/graphql-apollo-server-tutorial).
+- Recommendations use auto-magical Sequelize integration, which may not give us a solid boundary should DB technology change.
+  - Unsure if the driver would correctly handle [Aurora Serverless](research-AuroraServerless.md) errors.
+- Deployment is similarly undocumented. The [best source](https://levelup.gitconnected.com/deploying-an-apollo-graphql-application-as-an-aws-lambda-function-through-serverless-77fa33612bae) states outright that deployment is over-complicated, so developers should rely on "serverless" which we cannot use in our current architecture.
```

Comment:
we can use "AWS SAM"to deploy the serverless architecture if that is approved by NGAP.

Reply:
It is not the standard, currently all projects are being deployed via terraform or cloud formation with a terraform wrapper. We would want to keep this consistent for the end user.

Reply:
Are you referring to https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/what-is-sam.html ? That refers to programs that are serverless, not programs deployed using 'Serverless' https://www.serverless.com/framework/docs/providers/aws/guide/deploying/ .